static/js/settings_org.js

```diff
@@ -134,6 +137,46 @@ exports.render_notifications_stream_ui = function (stream_id) {
     elem.removeClass('text-warning');
 };
 
+function conversion_form_seconds(time_in_seconds) {
+    var unit_time_object = {
+        time_limit : 0,
+        unit_option_value : "",
+    };
+    if (time_in_seconds % (60*60*24) === 0) {
+        unit_time_object.time_limit = Math.ceil(time_in_seconds / (60*60*24));
+        unit_time_object.unit_option_value = "days";
+    } else if (time_in_seconds % (60*60) === 0) {
+        unit_time_object.time_limit = Math.ceil(time_in_seconds / (60*60));
+        unit_time_object.unit_option_value = "hours";
+    } else {
+        unit_time_object.time_limit = Math.ceil(time_in_seconds / (60));
+        unit_time_object.unit_option_value = "minutes";
+    }
+    return unit_time_object;
+}
+exports.update_message_edit_time_limit_settings = function (unit_time_object) {
+    // Updates the message_edit_limit_modal with appropriate values stored in unit_time_object
```

Comment:
I would do something like
`if settings_time_object is null, settings_time_object = convert_to_settings_format(DEFAULT_MESSAGE_CONTENT_EDIT_LIMIT_SECONDS)`

That will also be clear enough that we won't need a comment.